.github/workflows/test-e2e.yml

```diff
@@ -48,7 +48,7 @@ jobs:
 
       - name: Build docker image
         run: |
-          docker build --target test -t "ocelotsocialnetwork/webapp:test" webapp/
+          docker build --target test -t "ocelotsocialnetwork/webapp:test" --build-arg NODE_ENV=test webapp/
```

Comment:
This is good.

Maybe it does not the have the disired effect - at the moment the [webapp/.env.template](https://github.com/Ocelot-Social-Community/Ocelot-Social/blob/master/webapp/.env.template) is missing `NODE_ENV`.